todo_app/data/trello_items.py

```diff
@@ -0,0 +1,70 @@
+# create a class
+
+import requests
+import dotenv
+import os
+import json
+
+dotenv.load_dotenv("../.env")
+
+BOARD_ID = os.getenv("KAT_BOARD_ID")
+KEY = os.getenv("KAT_KEY")
+TOKEN = os.getenv("KAT_TOKEN")
+LIST_TODO = os.getenv("TODO_LIST_ID")
+LIST_DOING = os.getenv("DOING_LIST_ID")
+LIST_DONE = os.getenv("DONE_LIST_ID")
+
+
+
+##### Fetch functions
+
+
+def show_cards():
+    url = f"https://api.trello.com/1/boards/{BOARD_ID}/lists"
+
+    query = {"key": KEY, "token": TOKEN, "cards": "open"}
+    return requests.get(url, data=query).json()
+
+
+##### Add function
+
+
+def add_item(name):
+    url = "https://api.trello.com/1/card?idList={LIST_TODO}&key={KEY}&token={TOKEN}&name={name}"
+
+    query = {"idList": LIST_TODO, "key": KEY, "token": TOKEN, "name": name}
+    return requests.request("POST", url, data=query)
+
+
+##### Delete function
+
+
+def delete_item(id):
+    url = f"https://api.trello.com/1/cards/{id}?key={KEY}&token={TOKEN}"
+
+    query = {"id": id, "key": KEY, "token": TOKEN}
+    return requests.delete(url=url, data=query)
+
+
+##### Update function
+
+
```

Comment:
I'd argue that these comments aren't adding much over the names of your functions, which just leaves a second place that might end up out of sync with reality